salsa_dancing_molecules/worker_process/worker_process.py

```diff
@@ -16,7 +16,7 @@ def start(path):
     Args:
         path: The path to the working directory.
     """
-    print(path)
+    path = path.strip("/")
     print(path+"/unbegun_simulations")
```

Comment:
Is this print necessary?

Reply:
No, I forgot to remove it after using for testing